mods/force-chinese-ime.wh.cpp

```diff
@@ -0,0 +1,109 @@
+// ==WindhawkMod==
+// @id              force-chinese-ime
+// @name            Force Chinese IME Mode
+// @description     Forces Microsoft Pinyin IME to stay in Chinese mode
+// @version         1.0
+// @author          u3l6
+// @github          https://github.com/u3l6
+// @twitter         https://x.com/u_3l6
```

Comment:
Please send me (https://x.com/m417z) a direct message with the following content:

> I attest that I'm the sole owner of both this X account (https://x.com/u_3l6) and the following GitHub account: https://github.com/u3l6